test/langtools/jdk/javadoc/doclet/testSealedTypes/TestSealedTypes.java

```diff
@@ -254,50 +230,40 @@ public void testPartialMultiplePermits(Path base) throws IOException {
         javadoc("-d", base.resolve("out").toString(),
                 "--source-path", src.toString(),
                 "--no-platform-links",
-                "--enable-preview", "--source", thisRelease,
                 "p");
         checkExit(Exit.OK);
 
         checkOutput("p/A.html", true,
                 """
-                    <div class="type-signature"><span class="modifiers">public sealed<sup><a href="#\
-                    preview-p.A">PREVIEW</a></sup> class </span><span class="element-name type-name-\
-                    label">A</span>
-                    <span class="extends-implements">extends java.lang.Object</span>
-                    <span class="permits">permits<sup><a href="#preview-p.A">PREVIEW</a></sup> <a hr\
-                    ef="B.html" title="class in p">B</a><sup><a href="B.html#preview-p.B">PREVIEW</a\
-                    ></sup>, <a href="C.html" title="class in p">C</a><sup><a href="C.html#preview-p\
-                    .C">PREVIEW</a></sup> <span class="permits-note">(not exhaustive)</span></span><\
-                    /div>""");
+                <div class="type-signature"><span class="modifiers">public sealed class </span><span class="element-name type-name-label">A</span>
+                <span class="extends-implements">extends java.lang.Object</span>
+                <span class="permits">permits <a href="B.html" title="class in p">B</a>, <a href="C.html" title="class in p">C</a> <span class="permits-note">(not exhaustive)</span></span></div>
+                """);
     }
 
-    // @Test // javac incorrectly rejects the source
+    @Test
     public void testPartialMultiplePermitsWithSubtypes1(Path base) throws IOException {
         Path src = base.resolve("src");
         tb.writeJavaFiles(src,
                 "package p; public sealed class A permits B,C,D { }",
                 "package p; public final  class B extends A { }",
                 "package p; public final  class C extends A { }",
-                "package p;        sealed class D extends A permits D1, D2 { }",
+                "package p; sealed class D extends A permits D1, D2 { }",
```

Comment:
(minor)   don't delete the spaces; the class names were previously vertically aligned